packages/fxa-auth-server/test/local/routes/oauth.js

```diff
@@ -121,6 +124,27 @@ describe('/oauth/ routes', () => {
       });
       assert.deepEqual(resp, { key: 'data' });
     });
+
+    it('can refuse to return scoped-key data for selected OAuth clients', async () => {
+      mockOAuthDB = mocks.mockOAuthDB();
+      mockConfig.oauth.disableNewConnectionsForClients = [MOCK_CLIENT_ID];
+      sessionToken = await mockSessionToken();
+      const mockRequest = mocks.mockRequest({
+        credentials: sessionToken,
+        payload: {
+          client_id: MOCK_CLIENT_ID,
+          scope: MOCK_SCOPES,
+        },
+      });
+      try {
+        await loadAndCallRoute('/account/scoped-key-data', mockRequest);
+        assert.fail('should have thrown');
+      } catch (err) {
+        assert.equal(err.output.statusCode, 503);
```

Comment:
I'm used to seeing 503's representing server-side errors where a gateway failed, so the request should be retried as-is. Is this request going to work as-is later on its own?

Reply:
Yes, once we remove the temporarily-disabled client from the list.

Reply:
Do we include a `Retry-After` header automatically?

Reply:
I was about to say "yes we do" but it looks like this is not implemented on the `disabledClientId` error type; it is on some others such as `featureNotEnabled`: https://github.com/mozilla/fxa/blob/master/packages/fxa-auth-server/lib/error.js#L522

Reply:
I'm good either way, though for consistency it would probably be good to include it like the other 503's.

Reply:
I went ahead and added it.